projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -109,6 +109,26 @@ public function get_preferred_view( $screen, $fallback_global_preference = true
 		return parent::get_preferred_view( $screen, $fallback_global_preference );
 	}
 
+	/**
+	 * Adds Plugins menu.
+	 */
+	public function add_plugins_menu() {
+		/**
+		 * Whether to enable the marketplace feature entrypoint.
+		 * This filter is specific to WPCOM, that's why there is no
+		 * need to use `jetpack_` prefix.
+		 *
+		 * @use add_filter( 'wpcom_marketplace', '__return_true' );
+		 * @module masterbar
+		 * @since 10.3
+		 * @param bool $wpcom_marketplace Load the WordPress.com Marketplace feature. Default to false.
+		 */
+		if ( apply_filters( 'wpcom_marketplace', false ) ) {
```

Comment:
I think that filter may be a bit too generic. I can imagine other filters that may somehow interact with the Marketplace feature down the road, so I think it would make sense to be more specific. Even something like `wpcom_marketplace_enabled` would be a bit more specific I think. What do you think?

Reply:
We needed it to be generic so that we don't create new filters your suggestion `wpcom_marketplace_enabled` makes sense to me 109c3f9. Thanks!